src/DotLiquid.Website/ViewComponents/ContributorsViewComponent.cs

```diff
@@ -14,9 +13,9 @@ public class ContributorsViewComponent : ViewComponent
     {
         private readonly HttpClient _httpClient;
```

Comment:
This should be static. Even if the data is cached, I understand that each call to this component will instantiate a new `HttpClient`, which is incorrect.

Reply:
That was there before. However, now it is BETTER as this class is not even loaded because the view is cached and doesn't load it unnecessarily.

Reply:
I understand it was there before. Many things in this PR were there before and you still changed it. I didn't notice it was cached, cool. And I made the <cache> tag helper, at least I am not useless here ;)